src/config/schema.rs

```diff
@@ -289,6 +290,7 @@ impl Default for HttpFrontend {
             read_access: AccessSettings::Any,
             write_access: AccessSettings::Off,
             upload_data_max_length: 256,
+            cache_control: None,
```

Comment:
TBH we could make `max-age=3600, public`, or some other suitable value, the default (either here or by emitting it in the initial seafowl.toml that we write out on first startup).

Reply:
I went with a default here (12hours, as 1 hour seemed to small?).